example/geant4/CMakeLists.txt

```
@@ -56,15 +56,15 @@ celeritas_get_disable_device(CELER_DISABLE_DEVICE)
 
 #-----------------------------------------------------------------------------#
 
+set(_needs_g410)
+if(Geant4_VERSION VERSION_LESS 11.0)
+  set(_needs_g410 DISABLE)
+endif()
```

With the name of the variable I would expect:
```suggestion
set(_needs_g410)
if(Geant4_VERSION VERSION_LESS 10.0)
  set(_needs_g410 DISABLE)
endif()
```
or is the variable meant to be `_needs_g4_11_0` ?

---

Also https://github.com/celeritas-project/celeritas/pull/1925#discussion_r2317180120